src/zjs_ashell.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I wonder if these "virtual" modules could at least go in their own dir maybe? Just keep the ones that parallel real source files?